template/package.json

```diff
@@ -70,6 +67,7 @@
     "karma-spec-reporter": "0.0.31",
     "karma-webpack": "^2.0.2",
     "mocha": "^3.2.0",
+    "node-notifier": "^5.1.2",
```

Comment:
You should move this line outside of `unit` so it always gets installed

Reply:
Oh, right.  thanks!